tools/regz/src/gen.zig

```diff
@@ -824,10 +824,12 @@ fn write_fields(
         } else if (db.attrs.@"enum".get(fields[i].id)) |enum_id| {
             if (db.attrs.name.get(enum_id)) |enum_name| {
                 try writer.print(
-                    \\{}: packed union {{
-                    \\    raw: u{},
-                    \\    value: {},
-                    \\}},
+                // These were phased out in favor of using new builtins
+                //    \\{}: packed union {{
+                //    \\    raw: u{},
+                //    \\    value: {},
+                //    \\}},
+                    \\{}: {},
```

Comment:
You're going to have to remove `next.size` from the arguments now it's only printing two things.

Reply:
This was a terrible error. so embarrassing. I got the right number of items included now.